ppp_hal/requesthandler.py

```diff
@@ -40,22 +40,92 @@ def query(query, fields):
         mc.set(key, r, time=Config().memcached_timeout)
     return r
 
+PAPER_FIELDS = ('abstract_s', 'releasedDate_s', 'modifiedDateY_i',
+        'uri_s', 'halId_s', 'title_s', 'authFullName_s', 'arxivId_s',
+        'authFirstName_s', 'authLastName_s', 'authOrganism_s',
+        'labStructName_s', 'version_i', 'language_s')
+
+def graph_from_paper(paper):
+    same_as = list(filter(bool, (
+        paper['halId_s'],
+        paper.get('arxivId_s', None),
+        )))
+    organizations = paper.get('authOrganism_s', []) + \
+            paper.get('labStructName_s', [])
+    authors = [{'@context': 'http://schema.org/',
+                '@type': 'Person',
+                'name': fullname,
+                'givenName': firstname,
+                'familyName': lastname,
+               }
+               for (fullname, firstname, lastname) in
+               zip(paper['authFullName_s'], paper['authFirstName_s'],
+                   paper['authLastName_s'])]
+    d = {
+            '@type': 'ScholarlyArticle',
+            '@context': 'http://schema.org/',
```

Comment:
Please use "http://schema.org". I think that "http://schema.org/" is ok and works but is a bit confusing because it makes the context looks like an RDF namespace. It was a mistake in the spec. I've just updated it: https://github.com/ProjetPP/Documentation/commit/0c60de276170d023837235ce299e942a1aad08d1